src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -77,11 +78,14 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('repository', help='The repository name for a manifest-only copy of images. Multiple copies supported by passing --repository multiple times.', action='append')
         c.argument('force', help='Overwrite the existing tag of the image to be imported.', action='store_true')
 
-    with self.argument_context('acr config') as c:
+    with self.argument_context('acr config content-trust') as c:
         c.argument('status', help="Indicates whether content-trust is enabled or disabled.", arg_type=get_enum_type(PolicyStatus))
 
     with self.argument_context('acr config retention') as c:
+        c.argument('status', help="Indicates whether retention policy is enabled or disabled.", arg_type=get_enum_type(PolicyStatus))
```

Comment:
nit: probably can just say whether the policy is enabled to avoid defining it twice.

Reply:
You can also use a for loop like

```suggestion
        for cmd_name in ["content-trust", "retention"]:
           with self.argument_context('acr config {}'.format(cmd_name)) as c:
              c.argument('status', help="Indicates whether content-trust is enabled or disabled.", arg_type=get_enum_type(PolicyStatus)) 
```

This would be its own separate block of code from other argument registration such as registry_name

Reply:
@djyou modified string.
@adewaleo nice suggestion, but here we can keep it separate as the help message is different.